smart-contracts/.env.example

```diff
@@ -1,7 +1,7 @@
 # ------------
 #    General
 # ------------
-CONSENSUS_THRESHOLD=100
+CONSENSUS_THRESHOLD=25
```

Comment:
Could you please make a change to another .env file, but not this example file?

Reply:
Ok done